Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
+		{
+			get;
+			set;
+		} = 1000;
+
+		private TreeNodeAdv autoExpandNode = null;
+
+		private Timer timer = new Timer();
+
+		private void StartTimer()
+		{
+			if (DragOverExpandDelay > 0)
```

Comment:
Expanding on the above no-auto-properties comment: When there is no specific reason to do otherwise, please use the backing field directly inside your class. No need to invoke a getter method. This is for all occurrences of get-access of `DragOverExpandDelay`.